extensions/network/ping/SimplePing.m

```diff
@@ -577,7 +577,7 @@ static void SocketReadCallback(CFSocketRef s, CFSocketCallBackType type, CFDataR
 }
 
 /*! Starts the send and receive infrastructure.
- *  \details This is called once we've successfully resolved `hostName` in to
+ *  \details This is called once we've successfully resolved `hostName` into
```

Comment:
`in to`/`into` corrections have been quite contentious, but I'm reasonably confident about these corrections...